codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2StreamChannel.java

```diff
@@ -91,7 +94,7 @@ public boolean isOpen() {
 
     @Override
     public boolean isActive() {
-        return !closed;
+        return isOpen();
```

Comment:
Since reviewing this PR I also caught up on some of your discussion with @ejona86 ... staying consistent with HTTP handlers sounds reasonable. The definition of active for other connection oriented protocols is the channel becomes active once it is connected and you are able to send "data" ... where "data" would also include headers ... so sgtm.

However from a HTTP/2 perspective the `active` terminology is a bit confusing as it overlaps with the active streams semantics which carries different semantics.